src/common/utils.py

```diff
@@ -177,8 +177,8 @@ def get_s3_file_md5(bucket_name, key):
 """
 create error dict
 """
-def create_error(title, msg):
-    return {"title": title, "description": msg}
+def create_error(title, msg, code, severity, property_name, property_value):
```

Comment:
I suggest creating a centralized constants for error code and associated severity and title. So this function doesn't need title and severity parameters anymore, they can be found by code. So in the future, if we need to update title of severity of an error code, we only need to update the constants.

If you want to go even further, it maybe possible to store error message as a template in the code constants, and use given parameters (property_name, property_value, file_name, line_number etc.) to generate error message inside this function. But it's up to you to implement it or not.